lib/eslintBaseConfig.json

```diff
@@ -33,24 +35,34 @@
     "import/group-exports": "error",
     "import/no-default-export": "error",
     "import/no-unresolved": ["error", { "caseSensitiveStrict": true }],
-    "import/order": [
+    "import/order": "off",
+    "import/prefer-default-export": "off",
+    "no-console": "error",
+    "no-restricted-syntax": "off",
+    "no-use-before-define": ["error", { "functions": false }],
+    "perfectionist/sort-imports": [
       "error",
       {
+        "environment": "node",
         "groups": [
+          ["side-effect", "side-effect-style"],
           ["builtin", "external"],
           ["index", "internal", "parent", "sibling", "unknown"],
-          ["type"],
-          ["object"]
-        ],
-        "newlines-between": "always"
+          "object",
+          [
+            "builtin-type",
+            "external-type",
+            "index-type",
+            "internal-type",
+            "parent-type",
+            "sibling-type",
+            "type"
+          ],
+          "style"
+        ]
```

Comment:
a huge upside to these are they are now auto-fixable with `eslint-plugin-perfectionist`, so you can fix your whole codebase with one command (e.g. `eslint --fix`)